proto/cmp/services/transport/v1alpha1/search_result_types.proto

```diff
@@ -24,40 +24,41 @@ message TransportSearchResult {
   // Option to carry through an offer identifier for other stateful provider systems
   // like NDC AirShopping RS
   string offer_id = 2;
+  string segment_id = 3;
```

Comment:
This we already have under `TripSegment` message

Reply:
corrected